lib/spack/spack/test/cmd/env.py

```diff
@@ -924,29 +927,50 @@ def test_env_config_precedence(environment_from_manifest):
 """
         )
 
+    spack_yaml = tmp_path / ev.manifest_name
+    spack_yaml.write_text(
+        f"""\
+spack:
+  packages:
+    libelf:
+      version: ["0.8.12"]
+  include:
+  - {os.path.join(".", included_file)}
+  specs:
+  - mpileaks
+"""
+    )
+
+    e = ev.Environment(tmp_path)
     with e:
         e.concretize()
```

Comment:
We should probably standardize and either not use `with` here or not call it as `e.concretize()` as was fixed above.

Reply:
Fair point since I am modifying the test case.

Reply:
Actually, you need to call it with `e.concretize()` here if you want to use the method in the environment instance; otherwise, you're using the `concretize()` command.  There are other tests that exercise the latter so, from efficiency standpoint this is better.